mappings/net/minecraft/network/packet/s2c/play/PositionSeenS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_wgjrlqeo net/minecraft/network/packet/s2c/play/LookAtS2CPacket
+CLASS net/minecraft/unmapped/C_wgjrlqeo net/minecraft/network/packet/s2c/play/PositionSeenS2CPacket
```

Comment:
Doesnt feel right, dont know what to fix it to.

Reply:
hm, what feels wrong?
I like that name